src/app/app.component.ts

```diff
@@ -1,11 +1,39 @@
 import { Component, ViewEncapsulation } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { mergeAll } from 'rxjs';
+import { TableService } from './services/table.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
+  providers: [TableService]
 
 })
 export class AppComponent {
+
+  trElement:HTMLTableRowElement;
   title = 'angular-tour-of-heroes';
+  users = this.tableService.users;
+  id:number = -1;
+
+  constructor(private tableService: TableService) {
+
+  }
+
+  receiveTrElement(trElement:any): void {
+
+    this.id = parseInt(trElement.getAttribute('id'),10);
```

Comment:
ciekawe podejście, jakbyś mógł zrobić to inaczej? tak aby otrzymać już informacje jakie chcesz wyświetlić bez ponownego poszukiwania ich?